Makefile

```diff
@@ -73,12 +75,17 @@ version: .tag .commit .deployed
 	echo deployed $(BUILD_TIME)
 	sed $(SED_INPLACE) "s/BUILD_TIME/$(BUILD_TIME)/" $^
 	touch $@
-.appjson: $(DIST_DIR)/$(PACKAGE).json
+.appjson: $(DIST_DIR)/$(PACKAGE).json $(WHEELS)
 	echo appid: $(APP_ID)
 	echo name:  $(APP_NAME)
+	echo wheel: $(shell ls $(WHEELS))
 	sed $(SED_INPLACE) "s/APP_ID/$(APP_ID)/" $^
 	sed $(SED_INPLACE) "s/APP_NAME/$(APP_NAME)/" $^
 	sed $(SED_INPLACE) "s/MODULE/$(MODULE)/" $^
+	@echo "WHEELS: $(WHEELS)"
+# TODO: Add a SED command here for any custom library packaging.
+# sed $(SED_INPLACE) "s/WHEEL_TDX/$(shell ls $(WHEELS) | grep -E 'TDX.*\.whl')/" $@
```

Comment:
This line ensured that `app.json` has a reference to `TDXLib.0.8.1.whl`, rather than just a stub placeholder. Since the version number changes, this needs to be updated after each time we build the wheel file.

Reply:
DR: Wonder if we should write a little tool to do this, and include it in PhantomToolbox `phtoolbox`, and use it in the Makefile.

Reply:
As long as we're using Python for the tool, we could leverage Jinja2, which would make the un-filled `app.json` more readable, and hopefully make the whole expected replacements more clear at a glance